src/main/java/org/example/models/CurrencyExchange.java

```diff
@@ -0,0 +1,54 @@
+package org.example.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CurrencyExchange {
+
+    @JsonProperty("code")
+    private String code;
+
+    @JsonProperty("rates")
+    private List<CurrencyRate> rates;
+
+    private double bid;
+    private double ask;
+
+    public String getCode() {
+        return code;
+    }
+
+    public void setCode(String code) {
+        this.code = code;
+    }
+
+    public List<CurrencyRate> getRates() {
+        return rates;
+    }
+
+    public void setRates(List<CurrencyRate> currencyRates) {
+        this.rates = currencyRates;
+        if (!currencyRates.isEmpty()) {
```

Comment:
This logic should be moved up to json mapper if possible

Reply:
setRates logic moved to JSONMapper class